Add to docs (#66)
NASA-IMPACT/cloud-optimized-data-pipelines: stac-gen/README.md

> Docker to query CMR for meta data associated with a granule, create a STAC Item and then insert it into a remote database.

Reviewer: ```suggestion
This docker image queries CMR for metadata associated with a granule, creates a STAC Item and then inserts it into a remote database.
```
